services/order.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime | None = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket["movie_session"],
+                row=ticket["row"],
+                seat=ticket["seat"],
+                order=order,
+            )
+
+
+def get_orders(username: str | None = None) -> QuerySet:
+    orders = Order.objects.all()
+    if username:
+        user = get_user_model().objects.get(username=username)
+        orders = orders.filter(user=user)
```

Comment:
filter `orders` queryset that you assigned above with a lookup on user__username